pom.xml

```diff
@@ -96,6 +96,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <ddf.version>${project.version}</ddf.version>
         <!--Needed for documentation formatting-->
         <at-symbol>@</at-symbol>
```

Comment:
You might want to add that info that it is a workaround to the comment above it. That seems like something someone would "fix" by getting rid of it, thereby breaking the documentation.